### PR TITLE
Docs listening: Clarified telemetry opt-out processes

### PR DESCRIPTION
**Pull request by cwarnermm**
Increased visibility in the product docs that telemetry data collection is opt-out, as well as the steps to opt-out via the System Console. Included links to the individual configuration setting docs for additional reference.

### CODE REVIEW

File: source/manage/telemetry.rst [hunk 1]
```diff
@@ -9,12 +9,16 @@ As described in the privacy policy in each Mattermost server, telemetry data opt
...
-- To help improve the quality of Mattermost software and related services.
-- To make design decisions for future releases.
+- Identify security and reliability issues.
+- Analyze and fix software problems.
+- Help improve the quality of Mattermost software and related services.
+- Make design decisions for future releases.
+
+.. note::
+
+  Telemetry data collection is enabled by default for all Mattermost deployments. Self-hosted system admins can opt out of sharing telemetry data from Mattermost self-hosted servers within the System Console. Cloud system admins can't disable telemetry for Mattermost Cloud workspaces.
```
Comment (dontoisme) [inline source/manage/telemetry.rst:21]: Glad to see the callout for the difference between self-hosted and cloud.

**Review by dontoisme — APPROVED** [review]

**Comment by jordanafung** [comment]
I love the changes. Much clearer.
The only thing I would like to add is a link somewhere within the sidebar/menu.
What do we think?

**Comment by cwarnermm** [comment]
@jordanafung - You're correct that this page isn't accessible via the navigation pane. That's a historical choice worth revisiting.

@wiersgallak - Any concerns from your perspective about improving accessibility to this page by adding it to the nav pane?

**Comment by kwiersgalla** [comment]
I have no issues with moving it to the navigation, but I'll let @jasonblais also weigh in.

**Comment by jasonblais** [comment]
No concerns with having it in the navigation

File: source/manage/telemetry.rst [hunk 2]
```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
 - Date and time of the last check for security updates
 - The location of the Amazon Cloudfront server used for telemetry data
 
-To opt out, disable the feature in **System Console > Environment > SMTP**. When the feature is disabled, you will not receive any security alerts.
+Opt out
+~~~~~~~
+
+To opt out, you can disable this security update check feature for self-hosted deployments in the System Console by going to **Environment > SMTP > Enable Security Alerts**. See the `enable security alerts </configure/environment-configuration-settings.html#enable-security-alerts>`__ documentation for details. When this feature is disabled, you will not receive any security alerts.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:46]: We say "self-hosted servers" in the previous note but "self-hosted deployments" here. 

This is a great opportunity for me to standardize this provided we're referring to the same things. Or, if the server is different to the deployment (i.e., we're talking about discrete physical servers within a deployment of Mattermost that each need to be configured vs. the actual overarching Mattermost deployment) that's also OK. 

I'll work on the definitions/adding it to the list once we've decided on the terminology. :)

File: source/manage/telemetry.rst [hunk 3]
```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
...
 
 Error and diagnostics reporting feature
 ---------------------------------------
 
 Mattermost error and diagnostic data is collected for the following purposes:
+
 - To add improvements that are specific to your usage and deployment patterns, including identifying security and reliability issues.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:53]: I'm unsure of what a "deployment pattern" is - I don't think we use this term regularly. Do we mean the upgrade cadence?

File: source/manage/telemetry.rst [hunk 4]
```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
...
 - To analyze and fix software problems.
 - To help improve the quality of Mattermost software and related services.
 - To make design decisions for future releases.
 
 .. note:: 
 
-  Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. From Mattermost v5.23 the segment endpoint is being deprecated in favor of ``https://pdat.matterlytics.com``, a custom Rudder domain. To opt out, you can disable the feature in **System Console > Environment > Logging > Enable Diagnostics and Error Reporting**. Starting in Mattermost v5.26, when this feature is enabled, any 500 errors will be automatically sent to the Mattermost-hosted `Sentry <https://sentry.io/welcome/>`_ endpoint.
+  Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. From Mattermost v5.23, the segment endpoint is being deprecated in favor of ``https://pdat.matterlytics.com``, a custom Rudder domain. From Mattermost v5.26, when this feature is enabled, any 500 errors will be automatically sent to the Mattermost-hosted `Sentry <https://sentry.io/welcome/>`_ endpoint.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:60]: We don't use the word "client" often in our docs. Should we change this to "server"?

Comment (justinegeffen) [inline source/manage/telemetry.rst:60]:
```suggestion
  Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. From Mattermost v5.23, the segment endpoint is being deprecated in favor of ``https://pdat.matterlytics.com``, a custom Rudder domain. From Mattermost v5.26, when this feature is enabled, any 500 errors will automatically be sent to the Mattermost-hosted `Sentry <https://sentry.io/welcome/>`_ endpoint.
```

File: source/manage/telemetry.rst [hunk 5]
```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
...
+  
+Opt out
+~~~~~~~
+
+To opt out, you can disable the error and diagnostics reporting feature for self-hosted deployments in the System Console by going to **Environment > Logging > Enable Diagnostics and Error Reporting**. See the `enable diagnostics and error reporting </configure/environment-configuration-settings.html#enable-diagnostics-and-error-reporting>`__ documentation for details.
 
 Deployment and server configuration data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Reporting Frequency
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:70]:
```suggestion
Reporting frequency
```

File: source/manage/telemetry.rst [hunk 6]
```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
...
+
 - When starting the server for the first time: Every 10 minutes for the first hour, then every hour for the first 12 hours.
 - At the 24 hour mark and every 24 hours thereafter.
 
 Deployment Configuration Information
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:75]:
```suggestion
Deployment configuration information
```

File: source/manage/telemetry.rst [hunk 7]
```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
...
+
   Basic information including Mattermost server version, database and operating system type and version, and count of System Administrator accounts.
 
 Deployment Type
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:79]:
```suggestion
Deployment type
```

**Review by justinegeffen — COMMENTED** [review]
Apologies for the individual comments - I had more feedback than expected.

I didn't look at the whole document but in terms of the headers, if there's an opportunity to standardize them to sentence case that'd be great. Happy to help with this!

File: source/manage/telemetry.rst (hunk 3 above)
Comment (cwarnermm) [reply source/manage/telemetry.rst:53]: Great question, @justinegeffen 

While a deployment pattern is about changes in version control triggering a process to create deployable packages and run automated unit tests and code analysis, upgrade cadence is an organization's process around how often to review systems, networks, and applications for updates that remediate security vulnerabilities.

Given that this topic is about data telemetry gathering, I feel that both concepts apply. Thoughts?

File: source/manage/telemetry.rst (hunk 4 above)
Comment (cwarnermm) [reply source/manage/telemetry.rst:60]: I recommend a simplification from "client" to "Mattermost".

File: source/manage/telemetry.rst (hunk 3 above)
Comment (justinegeffen) [reply source/manage/telemetry.rst:53]: Yeah I agree. 

"To add improvements that are specific to your usage, upgrade, and deployment patterns, including identifying security and reliability issues."

Comment (cwarnermm) [inline source/manage/telemetry.rst:53]:
```suggestion
- To add improvements that are specific to your usage, upgrade, and deployment patterns, including identifying security and reliability issues.
```

Comment (cwarnermm) [reply source/manage/telemetry.rst]: Updated. Thank you!

**Comment by cwarnermm** [comment]
@jordanafung - You'll find the Telemetry page available from the navigation pane via Mattermost administration > Self-hosted administration > Manage telemetry.

A new generated preview of the docs will be available soon.

**Pull request merged**